scripts/tools/zap_regen_all.py

```diff
@@ -113,6 +144,39 @@ def generate(self) -> TargetRunStats:
         )
 
 
+class GoldenTestImageTarget():
+    def __init__(self):
+        # NOTE: output-path is inside the tree. This is because clang-format
+        #       will search for a .clang-format file in the directory tree
+        #       so attempts to format outside the tree will generate diffs.
+        os.makedirs('./out', exist_ok=True)
```

Comment:
Is this assuming this script is being run when cwd is top of tree?  If that's a new assumption, can we avoid making it (e.g. by doing this relative to the script location)?

Reply:
Main has a `os.chdir(CHIP_ROOT_DIR)`. All the paths seen in the script seem to be relative paths in general because of this.

It looks safe to rely on this.